### PR TITLE
Add a build step to test non freetype builds

### PR DESCRIPTION
**Pull request by nerzhul**
this permits to prevent regression for non freetype enabled builds

### CODE REVIEW

**Comment by Calinou** [comment]
Is there much point to supporting non-Freetype builds still? The font cannot be resized and doesn't support international characters as well as TrueType fonts do.

**Comment by nerzhul** [comment]
@Calinou excellent question, i always build with freetype and our windows build include freetype, android should have freetype too

**Comment by SmallJoker** [comment]
@Calinou Servers don't need freetype.

**Comment by nerzhul** [comment]
@SmallJoker valid point

**Comment by sfan5** [comment]
@SmallJoker servers don't need a GUI
`-DBUILD_SERVER=0` will disable all of that code

File: util/travis/script.sh [hunk 1]
```diff
@@ -22,6 +22,10 @@ if [[ ${PLATFORM} == "Unix" ]]; then
 		CMAKE_FLAGS+=' -DCUSTOM_GETTEXT_PATH=/usr/local/opt/gettext'
 	fi
 
+	if [[ ! -z ${FREETYPE} ]] && [[ "${FREETYPE}" == "0" ]]; then
```
Comment (sfan5) [inline util/travis/script.sh]: `[ -n "${FREETYPE}" ] &&` ...

**Pull request merged**